web/src/app/pages/create-survey/create-survey.component.ts

```diff
@@ -15,32 +15,31 @@
  */
 
 import {ActivatedRoute} from '@angular/router';
-import {Component, OnInit} from '@angular/core';
+import {Component, OnInit, ViewChild} from '@angular/core';
 import {SurveyService} from 'app/services/survey/survey.service';
-import {FormGroup, FormBuilder} from '@angular/forms';
+import {JobService} from 'app/services/job/job.service';
 import {NavigationService} from 'app/services/navigation/navigation.service';
+import {NameSurveyComponent} from 'app/pages/create-survey/name-survey/name-survey.component';
+import {NameJobComponent} from 'app/pages/create-survey/name-job/name-job.component';
 import {Survey} from 'app/models/survey.model';
+import {Job} from 'app/models/job.model';
 
 @Component({
   selector: 'create-survey',
   templateUrl: './create-survey.component.html',
   styleUrls: ['./create-survey.component.scss'],
 })
 export class CreateSurveyComponent implements OnInit {
-  readonly titleControlKey = 'title';
-  readonly descriptionControlKey = 'description';
-  private currentSurveyId: string | null = null;
-  formGroup: FormGroup;
+  currentSurveyId: string | null = null;
```

Comment:
Yes, we should use `string | undefined` more than `string | null`, but this is coming from `getSurveyId$` in `NavigationService`, and it has `| null` in a bunch of other places. We should clean this up later, it would be out of the scope of this PR.